Testscripts/Windows/VERIFY-HPC-VM.ps1

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+Check the VM has proper IB driver setup.
+
+.Description
+This test verified the HPC has IB over ND or SR-IOV.
+For general purpose VMs, this VM will abort.
+This script uses the function call, is_hpc_vm from utils.sh.
+This functions returns 0, if VM is IB over ND.
+Return 1, if VM is IB over SR-IOV.
+Return 2, if VM is non HPC VM.
+Return 3, if unknown.
+
+TODO: post-LIS installation in ND-based VM
+
+#>
+
+param([object] $AllVmData, [string]$TestParams)
+
+function Main {
+	param($AllVMData, $TestParams)
+	try {
+		#region Generate constants.sh
+		Write-LogInfo "Generating constants.sh ..."
```

Comment:
no need

Reply:
OK, I will use some param in future but can add this part later then.